source/spawn/spawn-channels.adb

```diff
@@ -710,19 +710,29 @@ package body Spawn.Channels is
            (Self.Stdin_Child,
             Self.Stdin_Parent,
             Success);
+
+      else
+         Self.Stdin_Parent := Glib.IOChannel.Ref (PTY_Channel);
       end if;
 
       if not Standard_Output_PTY then
```

Comment:
Please remove `not` and swap `then` and `else` parts.

Reply:
I disagree, main use case of "not PTY".